core-blocks/spacer/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WordPress
+ */
+import { __ } from '@wordpress/i18n';
+import ResizableBox from 're-resizable';
+import { InspectorControls } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+
+export const name = 'core/spacer';
+
+export const settings = {
+	title: __( 'Spacer' ),
+
+	description: __( 'Add an element with empty space and custom height.' ),
+
+	icon: 'image-flip-vertical',
+
+	category: 'layout',
+
+	attributes: {
+		height: {
+			type: 'number',
+			default: 100,
+		},
+	},
+
+	edit( { attributes, setAttributes, isSelected, toggleSelection } ) {
+		const { height } = attributes;
+
+		return [
+			<ResizableBox
+				size={ {
+					height,
+				} }
+				minHeight="20"
+				handleClasses={ {
+					top: 'wp-block-spacer__resize-handler-top',
+					bottom: 'wp-block-spacer__resize-handler-bottom',
+				} }
+				enable={ {
+					top: true,
```

Comment:
Sidenote: so bad API which enforces you to provide all those `false` values ...

Reply:
@gziolo yes :( At the very least we should abstract this with our own component. I also don't like the styles I had to supply.